CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
```

Comment:
Consider placing the `Salary` property after navigation and foreign key properties for better readability and consistent ordering as per established conventions.

```suggestion
public decimal Salary { get; set; }
```